app/javascript/controllers/flatpickr_controller.js

```diff
@@ -1,123 +1,28 @@
 import { Controller } from "@hotwired/stimulus"
 import flatpickr from "flatpickr"
 
-const invalidClass = "is-invalid"
-const timeValidator = /^[0-9]{4}-[0-9]{2}-[0-9]{2} [0-9][0-9]?:[0-9]{2}:[0-9]{2} (AM|PM)( [A-Z]+)?$/
 const dateValidator = /^[0-9]{4}-[0-9]{2}-[0-9]{2}$/
-const defaultHour = 12
 
 export default class extends Controller {
   connect() {
-    this.isTimestamp = !!this.element.dataset.timestamp
-    this.dateFormat = this.isTimestamp ? "Y-m-d h:i:S K" : "Y-m-d"
-    this.validator = this.isTimestamp ? timeValidator : dateValidator
-
-    // fix valueWas to match format (including timezone)
-    if (this.element.dataset.valueWas) {
-      const date = this.parseDate(this.element.dataset.valueWas)
-      const formatted = this.formatDate(date)
-      this.element.dataset.valueWas = formatted
-    }
-
-    // for timestamps, we need to make a hidden field for the actual ISO date
-    if (this.isTimestamp) {
-      const name = this.element.name
-      this.element.removeAttribute("name")
-      this.element.insertAdjacentHTML("afterend", `<input type="hidden" name="${name}">`)
-      this.hiddenField = this.element.nextSibling
-      this.hiddenField.value = this.element.value
-    }
-
-    this.picker = flatpickr(this.element, {
+    flatpickr(this.element, {
```

Comment:
so for the most part this controller is "gutted" so it has a fairly minimal amount of customization in it. this is still used in Podcast Planner? do we use (this version of) flatpickr anywhere else in Feeder? i know the dropdate picker also initializes flatpickr but it's a little differently used than here?

Reply:
Yep, the Planner still uses this.  We can revisit making this more customizable if we add more date/time fields to Feeder.  But for now, it seemed like all my hundreds of lines of custom code in here weren't helping things.